src/stories/Button.jsx

```diff
@@ -0,0 +1,50 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import './button.css';
+
+/**
+ * Primary UI component for user interaction
```

Comment:
These are example stories of storybook. You can delete them, because you don't need them to run storybook.